src/MacMSRDriver/DriverInterface.c

```diff
@@ -1,153 +1,80 @@
 // SPDX-License-Identifier: BSD-3-Clause
 // Copyright (c) 2012, Intel Corporation
 // written by Austen Ott
-//            
+//
 
 #include "DriverInterface.h"
 
 kern_return_t openMSRClient(io_connect_t connect)
 {
- 	kern_return_t	kernResult;
-	
-#if !defined(__LP64__)
-	if (IOConnectCallScalarMethod != NULL) {
-#endif
-		kernResult = IOConnectCallScalarMethod(connect, kOpenDriver, NULL, 0, NULL, NULL);
-#if !defined(__LP64__)
-	}
-	else {
-		kernResult = IOConnectMethodScalarIScalarO(connect, kOpenDriver, 0, 0);
-	}    
-#endif
-    
+        kern_return_t   kernResult;
+
+        kernResult = IOConnectCallScalarMethod(connect, kOpenDriver, NULL, 0, NULL, NULL);
 	return kernResult;
```

Comment:
should it return IOConnectCallScalarMethod(connect, kOpenDriver, NULL, 0, NULL, NULL); directly to simplify code?

Reply:
Done.